src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html

```diff
@@ -141,7 +143,7 @@
             <div class="left-editor">
                 <pre id="docEditor" class="form-control absolute-fill"
                      data-bind="aceEditor: { code: documentText, fontSize:'16px', lang: 'ace/mode/raven_document', readOnly: inReadOnlyMode() || inDiffMode() || isReadOnlyAccess() },
-                                disable: isBusy, validationOptions: { errorsAsTitle: false }, validationElement: documentText, visible: !isDeleteRevision()"></pre>
+                                disable: isBusy, validationOptions: { errorsAsTitle: false }, validationElement: documentText, visible: !isDeleteRevision() && !showHugeDocumentWarning()"></pre>
```

Comment:
please make sure ace editor doesn't get this huge doc at all.

Reply:
done